cmd/utils/flags.go

```diff
@@ -558,6 +558,28 @@ var (
 		Value:    ethconfig.Defaults.Miner.NewPayloadTimeout,
 		Category: flags.MinerCategory,
 	}
+	MevEnabledFlag = &cli.BoolFlag{
+		Name:     "mev.enable",
+		Usage:    "Enable mev",
+		Category: flags.MinerCategory,
+	}
+	SentryEnabledFlag = &cli.BoolFlag{
+		Name:     "sentry.enable",
+		Usage:    "Enable sentry",
+		Category: flags.MinerCategory,
+	}
+	SequencerUrl = &cli.StringFlag{
+		Name:     "sequencer.url",
+		Usage:    "Url of sequencer endpoint to use. Multiple urls are supported, separated by commas",
+		Value:    "http://127.0.0.1:8545",
```

Comment:
This default value is nonsense. I think we can remove it.

Reply:
Removed, please review again.